### ISSUE TITLE
[bug report] revertselectedRanges should work outside of diffEditor

### ISSUE THREAD

**Issue by xc1427** [kasecato/vscode-intellij-idea-keybindings#205]
In files `package-with-comment.json`, we now have

```
{
                "key": "ctrl+alt+z",
                "mac": "cmd+alt+z",
                "command": "git.revertSelectedRanges",
                "when": "isInDiffEditor",
                "intellij": "Rollback Lines"
            },
```
which specifies `isInDiffEditor`, but why not a more general case `editorTextFocus`.

**Comment by kasecato** [comment]
note
- https://github.com/microsoft/vscode/blob/1.52.1/extensions/git/package.json#L513
- https://github.com/microsoft/vscode/blob/1.52.1/extensions/git/src/commands.ts#L1026

**Comment by kasecato** [comment]
Hi, @xc1427  
I'll fix it. I went with the official key bindings, but I was wrong.
And I'll pull request it to VSCode repo, too.

Thank you.

**Comment by kasecato** [comment]
@xc1427 I released it on `v1.3.0`. Please check it

**Comment by kasecato** [comment]
Seems like VSCode doesn't change the default keymap, anymore.
